### PR TITLE
i18n(es): Update of `google-firebase.mdx`, `kinsta.mdx` & `images.mdx` 

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
 Updated different files based on these commits: 
   - `google-firebase.mdx` → 29aae1bdf726826b6effdc9aed7b142b7974e065
   - `kinsta.mdx` → fadae4274c54d07dd3be000aec28fb54fcc2cfcd
   - `images.mdx` → 8077640390594eab66c6497d5e9146c2e5bef371

### CODE REVIEW

File: src/content/docs/es/guides/deploy/kinsta.mdx [hunk 1]
```diff
@@ -31,12 +29,56 @@ Aquí están las líneas necesarias en tu archivo `package.json`:
...
 }
 ```
 
+### SSR
+:::tip [¿Buscas un ejemplo?]
+¡Echa un vistazo [al proyecto oficial de Kinsta Application Hosting Starter para Astro SSR](https://github.com/kinsta/hello-world-astro-ssr)!
+:::
+
+Para alojar tu proyecto en **Kinsta Application Hosting**, debes:
+- Incluye un campo `name` en tu `package.json`. (Esto puede ser cualquier cosa y no afectará su despliegue.)
```
Comment (Waxer59) [inline src/content/docs/es/guides/deploy/kinsta.mdx:44]:
```suggestion
- Incluir un campo `name` en tu `package.json`. (Esto puede ser cualquier cosa y no afectará su despliegue.)
```

File: src/content/docs/es/guides/deploy/kinsta.mdx [hunk 2]
```diff
@@ -31,12 +29,56 @@ Aquí están las líneas necesarias en tu archivo `package.json`:
...
+- Incluye un script `build` en tu `package.json`. (Tu proyecto de Astro ya debería incluir esto).
```
Comment (Waxer59) [inline src/content/docs/es/guides/deploy/kinsta.mdx:45]:
```suggestion
- Incluir un script `build` en tu `package.json`. (Tu proyecto de Astro ya debería incluir esto).
```

File: src/content/docs/es/guides/deploy/kinsta.mdx [hunk 3]
```diff
@@ -31,12 +29,56 @@ Aquí están las líneas necesarias en tu archivo `package.json`:
...
+- Instala el paquete [`@astrojs/node`](https://www.npmjs.com/package/@astrojs/node) y configura el script `start` en `node ./dist/server/entry.mjs`.
```
Comment (Waxer59) [inline src/content/docs/es/guides/deploy/kinsta.mdx:46]:
```suggestion
- Instalar el paquete [`@astrojs/node`](https://www.npmjs.com/package/@astrojs/node) y configura el script `start` en `node ./dist/server/entry.mjs`.
```

File: src/content/docs/es/guides/deploy/kinsta.mdx [hunk 4]
```diff
@@ -31,12 +29,56 @@ Aquí están las líneas necesarias en tu archivo `package.json`:
...
+- Configura `astro.config.mjs` para usar `@astrojs/node` y para usar `host: true`.
```
Comment (Waxer59) [inline src/content/docs/es/guides/deploy/kinsta.mdx:47]:
```suggestion
- Configurar `astro.config.mjs` para usar `@astrojs/node` y para usar `host: true`.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠 te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Gracias por la revisión @Waxer59, tenia en el olvido está PR 🫣

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the nice work @dreyfus92 & @Waxer59 💜

**Pull request merged**